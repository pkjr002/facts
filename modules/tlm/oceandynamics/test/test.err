tlm_fit_oceandynamics.py:209: RuntimeWarning: invalid value encountered in true_divide
  OceanDynTECorr = corr_num / corr_denom
Traceback (most recent call last):
  File "tlm_postprocess_oceandynamics.py", line 265, in <module>
    tlm_postprocess_oceandynamics(args.nsamps, args.seed, args.chunksize, args.keep_temp, args.pipeline_id)
  File "tlm_postprocess_oceandynamics.py", line 186, in tlm_postprocess_oceandynamics
    combined = xr.open_mfdataset("{0}_tempsamps_*.nc".format(pipeline_id), concat_dim="locations", chunks={"locations":chunksize})
  File "/home/rk509/.conda/envs/conda-entk/lib/python3.7/site-packages/xarray/backends/api.py", line 889, in open_mfdataset
    "When combine='by_coords', passing a value for `concat_dim` has no "
ValueError: When combine='by_coords', passing a value for `concat_dim` has no effect. To manually combine along a specific dimension you should instead specify combine='nested' along with a value for `concat_dim`.
mv: cannot stat ‘*localsl*’: No such file or directory
