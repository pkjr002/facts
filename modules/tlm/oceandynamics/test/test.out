Initiating test in /scratch/rk509/test.1666666936...
Extracting data files...
Executing workflow...
python tlm_preprocess_oceandynamics.py --pipeline_id oceandynamics-tlm-oceandynamics-ssp585 --scenario ssp585 --baseyear 2005 --pyear_start 2020 --pyear_end 2300 --pyear_step 10 --model_dir ./cmip6 --locationfile test/location_list.lst
python tlm_fit_oceandynamics.py --pipeline_id oceandynamics-tlm-oceandynamics-ssp585
python tlm_project_oceandynamics.py --pipeline_id oceandynamics-tlm-oceandynamics-ssp585 --nsamps 2000 --seed 4321 --tlm 1
python tlm_postprocess_oceandynamics.py --pipeline_id oceandynamics-tlm-oceandynamics-ssp585 --chunksize 662 --nsamps 2000 --seed 4321
Collecting output files...
